### PR TITLE
Update color-picker.tsx

### LINKED ISSUE #724: Some props can not be set to `ColorSelector` in `ColorPicker`
### Which package has an issue?

@yamada-ui/react

### Package Version

1.2.5

### Description

In `ColorSelector`, `channelProps` etc. are defined, but since props are not passed in `ColorPicker`, it cannot be customized.

### Link to Reproduction

_No response_

### Steps to Reproduce

1. Set `channelProps` to `ColorPicker`.
2. Can not be customized `ColorSelector` in `ColorPicker`.

### Browser

Chrome

### Operating System

- [X] macOS
- [X] Windows
- [X] Linux

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

`packages/components/color-picker/src/color-picker.tsx`

```diff
export type ColorPickerProps = ThemeProps<"ColorPicker"> &
  UseColorPickerProps &
  ColorPickerOptions
+ & Pick<
+   ColorSelectorProps,
+   | "withPicker"
+   | "withChannel"
+   | "saturationSliderRef"
+   | "saturationSliderProps"
+   | "swatchesProps"
+   | "withResult"
+   | "hueSliderRef"
+   | "hueSliderProps"
+   | "alphaSliderRef"
+   | "alphaSliderProps"
+   | "swatchesLabel"
+   | "swatches"
+   | "swatchesColumns"
+   | "channelProps"
+  >
```

```diff
    let {
      className,
      withEyeDropper = true,
      color,
      h,
      height,
      minH,
      minHeight,
      containerProps,
      inputProps,
      swatchProps,
      eyeDropperProps,
      portalProps = { isDisabled: true },
+     withPicker,
+     withChannel,
+     saturationSliderRef,
+     saturationSliderProps,
+     swatchesProps,
+      withResult,
+     hueSliderRef,
+     hueSliderProps,
+     alphaSliderRef,
+     alphaSliderProps,
+     swatchesLabel,
+     swatches,
+     swatchesColumns,
+     channelProps,
+     ...computedProps
    } = omitThemeProps(omitObject(mergedProps, ["withSwatch"]))
```

```diff
                <ColorSelector
                  className="ui-color-picker__picker"
+                 {...getSelectorProps({
+                   withPicker,
+                   withChannel,
+                   saturationSliderRef,
+                   saturationSliderProps,
+                   swatchesProps,
+                   withResult,
+                   hueSliderRef,
+                   hueSliderProps,
+                   alphaSliderRef,
+                   alphaSliderProps,
+                   swatchesLabel,
+                   swatches,
+                   swatchesColumns,
+                   channelProps,
+                 })}
                />
```

### PR DESCRIPTION
**Pull request by AllenWasupan**
Closes #724

## Description

> Add a brief description

## Current behavior (updates)

> Please describe the current behavior that you are modifying

## New behavior

> Please describe the behavior or changes this PR adds

### CODE REVIEW

**Comment by hirotomoyamada** [comment]
Please, create the file using the `pnpm changeset` command and see `contributing.md`.

**Comment by hirotomoyamada** [comment]
@AllenWasupan 

Hi, thanks for the PR!

Please, create the file using the `pnpm changeset` command and see `CONTRIBUTING.md`.

File: packages/components/color-picker/src/color-picker.tsx [hunk 1]
```diff
@@ -73,7 +73,23 @@ type ColorPickerOptions = {
...
+   | "withResult"
+   | "hueSliderRef"
+   | "hueSliderProps"
+   | "alphaSliderRef"
+   | "alphaSliderProps"
+   | "swatchesLabel"
+   | "swatches"
+   | "swatchesColumns"
+   | "channelProps"
+  >
```
Comment (KenyaMasuko) [inline packages/components/color-picker/src/color-picker.tsx:92]: The type `channelProps` isn't included in type `ColorSelectorProps` ( from `color-selector.tsx`), so you may want to consider augmenting the typing as follows in `color-selector.tsx`:
```tsx
  export type ColorSelectorProps = ThemeProps<"ColorSelector"> &
    UseColorSelectorProps &
    ColorSelectorOptions &
    Pick<
      ColorSelectorBodyProps,
      | "withResult"
      | "withEyeDropper"
      | "eyeDropperRef"
      | "eyeDropperProps"
      | "hueSliderRef"
      | "hueSliderProps"
      | "alphaSliderRef"
      | "alphaSliderProps"
    > &
    Pick<
      ColorSelectorSwatchesProps,
      "swatchesLabel" | "swatches" | "swatchesColumns" | "swatchProps"
    > &
    ColorSelectorChannelsProps // add this
```
The type `ColorSelectorChannelsProps` includes the `channelProps` property.

And in color-selector.tsx, ensure that `channelProps` are received via its props, and then pass `channelProps` to <ColorSelectorChannels /> like this: 
```tsx
const {
    className,
    withResult = true,
    withPicker = true,
    withChannel = true,
    swatches,
    swatchesColumns = 7,
    inputProps,
    withEyeDropper,
    eyeDropperRef,
    eyeDropperProps,
    saturationSliderRef,
    saturationSliderProps,
    hueSliderRef,
    hueSliderProps,
    alphaSliderRef,
    alphaSliderProps,
    swatchesLabel,
    swatchProps,
    swatchesProps,
    channelProps, // add this
    ...computedProps
} = omitThemeProps(mergedProps)

// other codes...

return (
  // other codes...

  {withPicker && withChannel ? (
    <ColorSelectorChannels channelProps={channelProps} /> // add `channelProps`
  ) : null}

  // other codes...
)
```

**Review by KenyaMasuko — CHANGES_REQUESTED** [review]
Hi, thank you for your contributing!

I've listed several points that need to be revised. Please check them out! Also, when making a pull request, please ensure to do so from the appropriate branch in accordance with the [contribution guide](https://github.com/hirotomoyamada/yamada-docs/blob/main/CONTRIBUTING.md#steps-to-pr)!

Keep it up💪

File: packages/components/color-picker/src/color-picker.tsx [hunk 2]
```diff
@@ -98,6 +114,20 @@ export const ColorPicker = forwardRef<ColorPickerProps, "input">(
...
+     swatchesProps,
+      withResult,
+     hueSliderRef,
+     hueSliderProps,
+     alphaSliderRef,
+     alphaSliderProps,
+     swatchesLabel,
+     swatches,
+     swatchesColumns,
+     channelProps,
```
Comment (umaidashi) [inline packages/components/color-picker/src/color-picker.tsx:130]: The indentation appears to be misaligned.

**Comment by hirotomoyamada** [comment]
@AllenWasupan 

This PR has duplicated corrections, so it will be closed.

**Pull request closed without merging**